Makefile

```diff
@@ -163,7 +169,7 @@ npm-refresh: ## Removes and rebuilds node app containers, images, volumes.
 
 db-update: ## Run the flyway database migration update (requires maven to be installed).
```

Comment:
You can now run the migration from the root of the repo without opening additional containers.